<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TxtEditor</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0">
    <item>
     <widget class="TabWidget" name="tabWidget" native="true"/>
    </item>
    <item>
     <widget class="SearchWidget" name="searchWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose_File"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionFind_Replace"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menu_Font_Size">
     <property name="title">
      <string>&amp;Font Size</string>
     </property>
     <addaction name="action2"/>
     <addaction name="action4"/>
     <addaction name="action6"/>
     <addaction name="action8"/>
     <addaction name="action10"/>
     <addaction name="action11"/>
     <addaction name="action12"/>
     <addaction name="action13"/>
     <addaction name="action14"/>
     <addaction name="action16"/>
     <addaction name="action18"/>
     <addaction name="action20"/>
     <addaction name="action22"/>
     <addaction name="action24"/>
     <addaction name="action26"/>
     <addaction name="action28"/>
     <addaction name="action30"/>
     <addaction name="action34"/>
     <addaction name="action38"/>
     <addaction name="action42"/>
     <addaction name="action48"/>
     <addaction name="action50"/>
     <addaction name="action60"/>
     <addaction name="action70"/>
     <addaction name="action72"/>
    </widget>
    <addaction name="menu_Font_Size"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_PSC_TxtEditor"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_File">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>&amp;New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>&amp;Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_File">
   <property name="icon">
    <iconset theme="document-close"/>
   </property>
   <property name="text">
    <string>&amp;Close File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="edit-undo"/>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="edit-redo"/>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionAbout_PSC_TxtEditor">
   <property name="text">
    <string>&amp;About TxtEditor</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset theme="document-save-as"/>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut"/>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy"/>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste"/>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>&amp;2</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>&amp;4</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>&amp;6</string>
   </property>
  </action>
  <action name="action8">
   <property name="text">
    <string>&amp;8</string>
   </property>
  </action>
  <action name="action10">
   <property name="text">
    <string>&amp;10</string>
   </property>
  </action>
  <action name="action11">
   <property name="text">
    <string>11</string>
   </property>
  </action>
  <action name="action12">
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="action13">
   <property name="text">
    <string>13</string>
   </property>
  </action>
  <action name="action14">
   <property name="text">
    <string>14</string>
   </property>
  </action>
  <action name="action16">
   <property name="text">
    <string>16</string>
   </property>
  </action>
  <action name="action18">
   <property name="text">
    <string>18</string>
   </property>
  </action>
  <action name="action20">
   <property name="text">
    <string>2&amp;0</string>
   </property>
  </action>
  <action name="action22">
   <property name="text">
    <string>22</string>
   </property>
  </action>
  <action name="action24">
   <property name="text">
    <string>24</string>
   </property>
  </action>
  <action name="action26">
   <property name="text">
    <string>26</string>
   </property>
  </action>
  <action name="action28">
   <property name="text">
    <string>28</string>
   </property>
  </action>
  <action name="action30">
   <property name="text">
    <string>&amp;30</string>
   </property>
  </action>
  <action name="action34">
   <property name="text">
    <string>34</string>
   </property>
  </action>
  <action name="action38">
   <property name="text">
    <string>38</string>
   </property>
  </action>
  <action name="action42">
   <property name="text">
    <string>42</string>
   </property>
  </action>
  <action name="action48">
   <property name="text">
    <string>48</string>
   </property>
  </action>
  <action name="action50">
   <property name="text">
    <string>&amp;50</string>
   </property>
  </action>
  <action name="action60">
   <property name="text">
    <string>60</string>
   </property>
  </action>
  <action name="action70">
   <property name="text">
    <string>&amp;70</string>
   </property>
  </action>
  <action name="action72">
   <property name="text">
    <string>72</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset theme="edit-select-all"/>
   </property>
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find/Replace</string>
   </property>
  </action>
  <action name="actionFind_Replace">
   <property name="icon">
    <iconset theme="edit-find-replace"/>
   </property>
   <property name="text">
    <string>&amp;Find/Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabWidget</class>
   <extends>QWidget</extends>
   <header>tabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchWidget</class>
   <extends>QWidget</extends>
   <header>searchwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
